src/_cffi_src/openssl/x509.py

```diff
@@ -65,6 +65,7 @@
 typedef struct {
     X509_CRL_INFO *crl;
     X509_ALGOR *sig_alg;
+    ASN1_BIT_STRING *signature;
```

Comment:
I don't think this should block, but we'll need to alter this for 1.1.0 using the new function `X509_CRL_get0_signature`. I'll deal with that in the 1.1.0 WIP PR though.